snuba/query/processors/tagsmap.py

```diff
@@ -103,6 +104,14 @@ def __is_optimizable(
             )
         return None
 
+    def __find_tags(self, expression: Expression) -> bool:
```

Comment:
I'd expect a `find` method to return the thing it found, like `string.find` or `re.findall`. Maybe this should be `__has_tags`?

Reply:
yeah, makes sense, will rename